src/skills-builder/skills-builder-modal/view-results/RelatedSkillsSelectableBoxSet.jsx

```diff
@@ -27,7 +26,8 @@ const RelatedSkillsSelectableBoxSet = ({ jobSkillsList, selectedJobTitle, onChan
       type="radio"
       value={selectedJobTitle}
       onChange={onChange}
-      columns={isExtraSmall ? 1 : 3}
+      columns={3}
+      className="overflow-scroll-medium"
```

Comment:
The figma also mentions adding a label with "Select job title to see recommended courses". Is that an aria-label? Is that added to the SelectableBox?

Reply:
That sounds non-trivial. Let's make another ticket